sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/MainActivity.java

```diff
@@ -35,6 +41,18 @@ protected void onCreate(Bundle savedInstanceState) {
           Sentry.captureUserFeedback(userFeedback);
         });
 
+    binding.sendAttachment.setOnClickListener(
+        view -> {
+          Thread thread =
+              new Thread() {
+                public void run() {
+                  captureExceptionWithAttachments();
+                }
+              };
+          // Fire and forget
+          thread.start();
```

Comment:
L: why do we use a Thread?

Reply:
Because writing a 2 MB file blocks the UI thread for a bit. Would you prefer to introduce a proper way of running tasks in the background, like `ExecutorService`, Coroutines, Rx, ...?

Reply:
mm, I'd say adding a comment would be good enough, just to not bound that attachments feature 'require' a Thread or so, I mean, it's highly recommended, but not really needed

Reply:
Ah, now I get your point. Didn't think about users might look this up and think we need a thread for this. I'm going to add a comment.